data-service-model/src/main/java/com/amazonaws/blox/dataservicemodel/v1/exception/ResourceInUseException.java

```diff
@@ -22,7 +24,11 @@
   private String resourceType;
   private String resourceId;
 
-  public ResourceInUseException(String resourceType, String resourceId, String message) {
+  @JsonCreator
+  public ResourceInUseException(
+      @JsonProperty("resourceType") String resourceType,
+      @JsonProperty("resourceId") String resourceId,
+      @JsonProperty("message") String message) {
```

Comment:
Not in this PR, but should we build this exception same as `ResourceExistsException`, only use `resourceType` and `resourceId` but the message is part of the exception logic.

Reply:
Absolutely, in a followup PR.